app/models/notification.rb

```diff
@@ -0,0 +1,41 @@
+class Notification < ActiveRecord::Base
```

Comment:
I'm not sure if there ever was a design for this, but from I talked with joao, it seems this was the desired functionality (at least for him, it's possible this wasn't discussed globally)

All notifications would have nothing more than a text message, and a url, and they would show up somewhere on the front end (the sidebar possibly? Clicking on a notification should redirect you to that URL, and delete the notification itself